roles/mongodb/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+galaxy_info:
+  description: Install MongoDB-3.2
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: CentOS/RedHat
```

Comment:
I surfed a bit through other galaxy roles and looks like they just write `EL`.

See https://github.com/geerlingguy/ansible-role-postgresql/blob/master/meta/main.yml